### PR TITLE
update to latest test server and regenerate code

### PR DESCRIPTION
**Pull request by jhendrixMSFT**

### CODE REVIEW

File: src/generator/operations.ts [hunk 1]
```diff
@@ -438,6 +447,14 @@ function createProtocolRequest(codeModel: CodeModel, client: string, op: Operati
         text += '\tunencodedParams := []string{}\n';
       }
       for (const qp of values(unencodedParams)) {
+        let setter: string;
+        if (qp.protocol.http?.explode === true) {
+          setter = `\tfor _, qv := range ${getParamName(qp)} {\n`;
+          setter += `\t\tunencodedParams = append(unencodedParams, "${qp.language.go!.serializedName}="+qv)\n`;
```
Comment (catalinaperalta) [inline src/generator/operations.ts:453]: Just want to know what the = for after `${qp.language.go!.serializedName}`? I dont think I saw it in any of the testserver examples.

File: src/generator/operations.ts [hunk 2]
```diff
@@ -426,7 +426,16 @@ function createProtocolRequest(codeModel: CodeModel, client: string, op: Operati
     if (encodedParams.length > 0) {
       text += '\tquery := u.Query()\n';
       for (const qp of values(encodedParams)) {
-        text += emitQueryParam(qp, `query.Set("${qp.language.go!.serializedName}", ${formatParamValue(qp, imports)})`);
+        let setter: string;
+        if (qp.protocol.http?.explode === true) {
```
Comment (catalinaperalta) [inline src/generator/operations.ts:430]: For my own knowledge, what does explode mean for us in swagger?

Comment (jhendrixMSFT) [reply src/generator/operations.ts:430]: It comes from https://swagger.io/specification/.  Unfortunately I can't link to the exact section but if you search for "explode" it's the first hit.

File: src/generator/operations.ts (hunk 1 above)
Comment (jhendrixMSFT) [reply src/generator/operations.ts:453]: When exploding a query param `queryParam` with the following values `foo,bar,baz` we want to generate the following sequence.

`queryParam=foo&queryParam=bar&queryParam=baz`

You are right there's no codegen with this as we don't have a test server example that uses exploded, unencoded query params.

Comment (catalinaperalta) [reply src/generator/operations.ts:453]: Ok got it 👍 yeah we should have something that tests this in particular

File: src/generator/operations.ts (hunk 2 above)
Comment (catalinaperalta) [reply src/generator/operations.ts:430]: got it, thanks for the info!

**Pull request merged**